### DISCUSSION TITLE
Common limit for all endpoints

### DISCUSSION THREAD

**Discussion by YuriiMotov** [laurentS/slowapi#175] [Q&A] [answered]
Can I limit the total amount of requests for particular user per minute (it doesn't matter what endpoints does this person use, I need to limit total amount of request for this person)?

**Comment by ecly** [comment, marked as answer]
@YuriiMotov yes, this is exactly what the `application_limits` option for the [`Limiter`](https://slowapi.readthedocs.io/en/latest/api/) class allows.
